qtfred/src/ui/widgets/sexp_tree.cpp

```diff
@@ -5017,6 +5027,22 @@ sexp_list_item *sexp_tree::get_listing_opf_sexp_containers(ContainerType con_typ
 	return head.next;
 }
 
+sexp_list_item* sexp_tree::get_listing_opf_asteroid_debris()
```

Comment:
One minor quibble... this function in FRED2 is between `get_listing_opf_nebula_patterns` and `get_listing_opf_game_snds`.  Wherever the function might happen to be, it should be in the same relative location in FRED and qtFRED to make it easier to compare the two codebases.

(Keeping this function where it is and moving the FRED2 function is a potential solution.)